src/Performer/Performer.jsx

```diff
@@ -4,42 +4,71 @@ import config from '../../env/config.js';
 import NewPerformance from './Components/NewPerformance.jsx';
 import PerformerInfo from './Components/PerformerInfo.jsx';
 import PerformerTile from './Components/PerformerTile.jsx';
+import Feed from '../Shared/Feed.jsx';
 
 const Performer = () => {
-  const [latLng, setLatLng] = useState();
-  const performances = [1, 2, 3];
-  const locality = 'Boston MA';
-  const country = 'United States';
+  const [latLng, setLatLng] = useState(null);
+  const [profile, setProfile] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const buskerName = 'Shrek'
 
   // sets the default coordinates to the area the busker is located in
-  const setMapArea = () => {
-    const configAxios = {
+  const setMapArea = (object) => {
+    const locality = object.Location.split(',')[0]
+    const country = object.Location.split(',')[1]
+    const profileMap = {
       method: 'get',
       url: `https://maps.googleapis.com/maps/api/geocode/json?components=locality:${locality}|country:${country}&key=${config.googleMaps.API}`,
-      headers: { },
+      headers: {},
     };
 
-    axios(configAxios)
+    axios(profileMap)
       .then((response) => {
         setLatLng(response.data.results[0].geometry.location);
       })
       .catch((error) => {
         console.log(error);
       });
   };
+  // gets busker profile
+  const getBuskerProfile = () => {
+    const configProfile = {
+      method: 'get',
+      url: `http://localhost:3000/profile/${buskerName}/busker`,
+      headers: {},
+    };
+
+    axios(configProfile)
+      .then((response) => {
+        setProfile(response.data[0]);
+        if (latLng === null) {
+          setMapArea(response.data[0]);
+        }
+        setIsLoading(false);
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+
+  }
 
   useEffect(() => {
-    setMapArea();
+    getBuskerProfile();
   }, []);
 
+  if (isLoading) {
+    return <div>Loading</div>;
+  }
+
   return (
     <div>
-      <PerformerInfo />
-      <NewPerformance latLng={latLng} />
-      {performances.map(() => <PerformerTile />)}
+      <PerformerInfo profile={profile}/>
+      <NewPerformance getBuskerProfile={getBuskerProfile} profile={profile} latLng={latLng} />
+      {/* <Feed performances={profile.Events}/> */}
+      {profile.Events.map((event) => <PerformerTile event={event} />)}
```

Comment:
Please add a key to the PerformerTile component to reduce errors in the console. Key can be anything so long as it will be unique to each component.